client/homescreen/activity-panel/style.scss

```diff
@@ -1,67 +1,70 @@
-@mixin accordion-header {
-	.woocommerce-accordion-header {
-		order: 1;
-		text-align: left;
-		.woocommerce-accordion-title {
-			margin-right: $gap;
-		}
-	}
-}
-.woocommerce-accordion-card {
-	.components-card__header,
-	.components-panel__body-title .components-button {
-		font-size: 20px;
-		line-height: 28px;
-		color: $gray-900;
-		&:focus {
-			box-shadow: unset;
-			outline: unset;
+.woocommerce-activity-panel {
+	background: transparent;
+	border: 0;
+
+	.components-panel__body {
+		background: #fff;
+		border: 1px solid $gray-200;
+
+		&.is-opened {
+			margin-bottom: $gap-large;
+
+			.components-panel__body-toggle {
+				border-bottom: 1px solid $gray-200;
+			}
 		}
 	}
 
-	.components-card__header {
-		height: 60px;
-		@include accordion-header();
+	&.components-panel > .components-panel__body:last-child {
+		border-bottom: 1px solid $gray-200;
 	}
 
-	.components-panel__body-title {
-		margin: 0;
-		padding: $gap-small 0;
-		height: 60px;
-		.components-button {
+	.components-panel__row {
+		margin: 0 -#{$gap} -#{$gap} -#{$gap};
+		> div {
```

Comment:
Fwiw this is the kind of style that can break based on different versions of `@wordpress/components`. I understand sometimes its impossible to avoid, but is there a way you can avoid relying on immediate child div here?

Reply:
Thats all good 👍